src/Form/matching-config/selectors-css.js

```diff
@@ -2,9 +2,13 @@
 const genericTextField = `
 input:not([type=button]):not([type=checkbox]):not([type=color]):not([type=file]):not([type=hidden]):not([type=radio]):not([type=range]):not([type=reset]):not([type=image]):not([type=search]):not([type=submit]):not([type=time]):not([type=url]):not([type=week]):not([name^=fake i]):not([data-description^=dummy i]):not([name*=otp]):not([autocomplete="fake"]):not([placeholder^=search i]):not([type=date]):not([type=datetime-local]):not([type=datetime]):not([type=month])`
 
-const formInputsSelector = [
+const formInputsSelectorWithoutSelect = [
     genericTextField,
-    '[autocomplete=username]',
+    '[autocomplete=username]'
+]
+
+const formInputsSelector = [
+    ...formInputsSelectorWithoutSelect,
```

Comment:
Separated the one with `select` and the one without. This allows us to skip `select` in the entry point scan, so if a page or a form only has a `select` element we don't waste any resources scanning something we can't fill anyway.